### ISSUE TITLE
Git: Revert cloning of other's repo

### ISSUE THREAD

**Issue by huangzz125** [nus-cs2103-AY2223S2/forum#138]
<img width="1165" alt="image" src="https://user-images.githubusercontent.com/77564512/216751913-fd0b4a89-91bd-43b2-8a67-12213908daa3.png">

I have accidentally cloned Tan Jun Da's repo when I was trying to review his PR, I am not sure how I can undo this. Any help will be appreciated. Thanks!

**Comment by francisyzy** [comment]
You may try to push your changes to GitHub, afterwards delete your local folder and clone your own repo again? To be safe, I would backup the local folder first before attempting this to ensure I do not delete any important files.

**Comment by damithc** [comment]
> I have accidentally cloned Tan Jun Da's repo

@huangzz125 cloned or just fetched a branch from his repo? cloning gives you a separate repo; it doesn't add commits to your existing repo.

If it was just fetching the branch, just delete it locally and things should be back to normal.

**Comment by domlimm** [comment]
Hello @huangzz125, I'm a senior from a previous batch, dropping by to look at the issues posted by you guys. Just a gentle reminder to close the issue once your problem has been resolved. Have a great day! 😄

**Comment by EvitanRelta** [comment]
Based on ur Git/SourceTree graph, it looks like you've fetched the `A-Enums` tag from their repo.  
It doesn't seem like u pulled/fetched any branches.

Then:

1. As @francisyzy suggested, using ur file explorer, you should <ins>**FIRST copy your entire repo folder**</ins> to back it up.

2. I'm not too sure what GUI app u're using, 
   but u should be able to right-click Jun Da's `A-Enums` and delete it.

3. That's should fix the problem in ur screenshot. 
   But do post more info/pictures if u actually clone/pull/fetched more then just that `A-Enums` tag.

**Comment by damithc** [comment]
> Based on ur Git/SourceTree graph, it looks like you've fetched the `A-Enums` tag from their repo.
> It doesn't seem like u pulled/fetched any branches.

✔️

**Comment by huangzz125** [comment]
Initially, I tried removing that tag locally by running `git tag -d A-Enums` but it still results in that tag re-appearing on both my Sourcetree and when I run `git tag -l` on terminal a few seconds after it was successfully deleted (`Deleted tag 'A-Enums' (was 5ab0c61)` was displayed). So, I thought that the tag on my remote somehow. 

I could not remember what I did that caused that tag to end up being kept track by my Git but does this mean that I have pulled it and pushed it to my own remote?

So, I ran `git push --delete origin A-Enums` on top of the command to delete the tag locally. Now, the tag is completely deleted.

Thanks @EvitanRelta , point 2 worked perfectly. Thanks Prof @damithc and other friends for your kind help.
